### ISSUE TITLE
EXCEPTION_ACCESS_VIOLATION crash when starting level

### ISSUE THREAD

**Issue by jaydenmilne** [BYU-PCCL/holodeck#270]
**Describe the bug**
Sometimes when starting a level, the json parser throws an EXCEPTION_ACCESS_VIOLATION exception. This might be solved by doing #266 

**To Reproduce**
Run `test_loading_scenarios.py` 

**Log**
```
[2019.05.30-18.52.42:909][  5]LogWindows: Error: === Critical error: ===
[2019.05.30-18.52.42:909][  5]LogWindows: Error: 
[2019.05.30-18.52.42:909][  5]LogWindows: Error: Fatal error!
[2019.05.30-18.52.42:909][  5]LogWindows: Error: 
[2019.05.30-18.52.42:909][  5]LogWindows: Error: Unhandled Exception: EXCEPTION_ACCESS_VIOLATION reading address 0x00000000
[2019.05.30-18.52.42:909][  5]LogWindows: Error: 
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff64db644d6 Holodeck.exe!jsonParse() [c:\users\jayde\documents\holodeck-engine\source\holodeck\utils\private\gason.cpp:169]
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff64db60a44 Holodeck.exe!UCommandCenter::Tick() [c:\users\jayde\documents\holodeck-engine\source\holodeck\clientcommands\private\commandcenter.cpp:22]
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff64db603c6 Holodeck.exe!AHolodeckGameMode::Tick() [c:\users\jayde\documents\holodeck-engine\source\holodeck\holodeckcore\private\holodeckgamemode.cpp:25]
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff64fe2e296 Holodeck.exe!AActor::TickActor() []
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff64fe01f30 Holodeck.exe!FActorTickFunction::ExecuteTick() []
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff650b84ce5 Holodeck.exe!FTickFunctionTask::DoTask() []
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff650b9c906 Holodeck.exe!TGraphTask<FTickFunctionTask>::ExecuteTask() []
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff64dbb1376 Holodeck.exe!FNamedTaskThread::ProcessTasksNamedThread() []
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff64dbb1896 Holodeck.exe!FNamedTaskThread::ProcessTasksUntilQuit() []
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff650be9d21 Holodeck.exe!FTickTaskSequencer::ReleaseTickGroup() []
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff650bf8fb8 Holodeck.exe!FTickTaskManager::RunTickGroup() []
[2019.05.30-18.52.42:909][  5]LogWindows: Error: [Callstack] 0x00007ff6504c1eaf Holodeck.exe!UWorld::RunTickGroup() []
[2019.05.30-18.52.42:910][  5]LogWindows: Error: [Callstack] 0x00007ff6504cad6f Holodeck.exe!UWorld::Tick() []
[2019.05.30-18.52.42:910][  5]LogWindows: Error: [Callstack] 0x00007ff650328ed9 Holodeck.exe!UGameEngine::Tick() []
[2019.05.30-18.52.42:910][  5]LogWindows: Error: [Callstack] 0x00007ff64d6008d9 Holodeck.exe!FEngineLoop::Tick() []
[2019.05.30-18.52.42:910][  5]LogWindows: Error: [Callstack] 0x00007ff64d61032c Holodeck.exe!GuardedMain() []
[2019.05.30-18.52.42:910][  5]LogWindows: Error: [Callstack] 0x00007ff64d61038a Holodeck.exe!GuardedMainWrapper() []
[2019.05.30-18.52.42:910][  5]LogWindows: Error: [Callstack] 0x00007ff64d61e975 Holodeck.exe!WinMain() []
[2019.05.30-18.52.42:910][  5]LogWindows: Error: [Callstack] 0x00007ff6519bd9a6 Holodeck.exe!__scrt_common_main_seh() [d:\agent\_work\1\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288]
[2019.05.30-18.52.42:910][  5]LogWindows: Error: [Callstack] 0x00007ffa38f44034 KERNEL32.DLL!UnknownFunction []
[2019.05.30-18.52.42:910][  5]LogWindows: Error: [Callstack] 0x00007ffa39b03691 ntdll.dll!UnknownFunction []
```

**Comment by jaydenmilne** [comment]
Note that this bug is intermittent, so it might be a race somewhere.

**Comment by jaydenmilne** [comment]
Fixed in BYU-PCCL/holodeck-engine#175
